### DISCUSSION TITLE
Toggle for disabling "Loading - please wait freeze"

### DISCUSSION THREAD

**Discussion by tannerdino** [runelite/runelite#16708] [Ideas]
The Map loading improvements update for runelite should be a toggle. Many people prefer the old map loading, thanks :)

**Comment by Nightfirecat** [comment]
https://xkcd.com/1172/ ?
I'm a little confused, why would you or anyone else prefer the old--objectively worse--map loading?

**Comment by KevzaraOSRS** [comment]
tbh since this update came out, my runelite will freeze while loading chunks in CoX and will generally take slower to load each raid while scouting. Would be super nice to be able to toggle that off until those two things are fixed.

**Reply by Felanbird** [reply]
Turn off 3d-weather

**Reply by KevzaraOSRS** [reply]
holy shit thank you
